CHANGELOG.md

```diff
@@ -6,11 +6,79 @@ document.
 
 ## Unreleased / In Rust Nightly
 
-[74d1561...master](https://github.com/rust-lang/rust-clippy/compare/74d1561...master)
+[7bfc26e...master](https://github.com/rust-lang/rust-clippy/compare/7bfc26e...master)
+
+## Rust 1.56
+
+Current beta, release 2021-10-21
+
+[74d1561...7bfc26e](https://github.com/rust-lang/rust-clippy/compare/74d1561...7bfc26e)
+
+### New Lints
+
+* [`unwrap_or_else_default`]
+  [#7516](https://github.com/rust-lang/rust-clippy/pull/7516)
+
+### Enhancements
+
+* [`needless_continue`] now also lints in `loop { continue; }` case
+  [#7477](https://github.com/rust-lang/rust-clippy/pull/7477)
```

Comment:
The previous versions used a colon after the lint name and then continued with a capital letter. I would suggest using the same format for consistency. The same goes for the other messages as well. What are your thoughts about it? :upside_down_face: 
```suggestion
* [`needless_continue`]: now also lints in `loop { continue; }` case
  [#7477](https://github.com/rust-lang/rust-clippy/pull/7477)
```

Reply:
I seem to have forgotten my own formatting here :smile: Will change that.